test/functions.test.js

```diff
@@ -1,130 +1,488 @@
+      Genderqueer: 1,
+      Male: 23,
+      "Non-binary": 1,
+    },
+    Eagle: {
+      Bigender: 1,
+      Female: 19,
+      Genderqueer: 1,
+      Male: 23,
+      "Non-binary": 1,
+    },
+    Ford: {
+      Bigender: 1,
+      Female: 19,
+      Genderqueer: 1,
+      Male: 23,
+      "Non-binary": 1,
+    },
+    GMC: {
+      Bigender: 1,
+      Female: 19,
+      Genderqueer: 1,
+      Male: 23,
+      "Non-binary": 1,
+    },
+    Infiniti: {
+      Bigender: 1,
+      Female: 19,
+      Genderqueer: 1,
+      Male: 23,
+      "Non-binary": 1,
+    },
+    Jeep: {
+      Bigender: 1,
+      Female: 19,
+      Genderqueer: 1,
+      Male: 23,
+      "Non-binary": 1,
+    },
+    Kia: {
+      Bigender: 1,
+      Female: 19,
+      Genderqueer: 1,
+      Male: 23,
+      "Non-binary": 1,
+    },
+    Lexus: {
+      Bigender: 1,
+      Female: 19,
+      Genderqueer: 1,
+      Male: 23,
+      "Non-binary": 1,
+    },
+    Mazda: {
+      Bigender: 1,
+      Female: 19,
+      Genderqueer: 1,
+      Male: 23,
+      "Non-binary": 1,
+    },
+    "Mercedes-Benz": {
+      Bigender: 1,
+      Female: 19,
+      Genderqueer: 1,
+      Male: 23,
+      "Non-binary": 1,
+    },
+    Mercury: {
+      Bigender: 1,
+      Female: 19,
+      Genderqueer: 1,
+      Male: 23,
+      "Non-binary": 1,
+    },
+    Mitsubishi: {
+      Bigender: 1,
+      Female: 19,
+      Genderqueer: 1,
+      Male: 23,
+      "Non-binary": 1,
+    },
+    Oldsmobile: {
+      Bigender: 1,
+      Female: 19,
+      Genderqueer: 1,
+      Male: 23,
+      "Non-binary": 1,
+    },
+    Peugeot: {
+      Bigender: 1,
+      Female: 19,
+      Genderqueer: 1,
+      Male: 23,
+      "Non-binary": 1,
+    },
+    Pontiac: {
+      Bigender: 1,
+      Female: 19,
+      Genderqueer: 1,
+      Male: 23,
+      "Non-binary": 1,
+    },
+    Porsche: {
+      Bigender: 1,
+      Female: 19,
+      Genderqueer: 1,
+      Male: 23,
+      "Non-binary": 1,
+    },
+    Scion: {
+      Bigender: 1,
+      Female: 19,
+      Genderqueer: 1,
+      Male: 23,
+      "Non-binary": 1,
+    },
+    Subaru: {
+      Bigender: 1,
+      Female: 19,
+      Genderqueer: 1,
+      Male: 23,
+      "Non-binary": 1,
+    },
+    Toyota: {
+      Bigender: 1,
+      Female: 19,
+      Genderqueer: 1,
+      Male: 23,
+      "Non-binary": 1,
+    },
+    Volvo: {
+      Bigender: 1,
+      Female: 19,
+      Genderqueer: 1,
+      Male: 23,
+      "Non-binary": 1,
+    },
+  };
+
+  const actual = getGenderBreakdownOfEachCar(customers);
+
+  expect.deepEqual(actual, expected);
 });
 
-skip('getAllCoolFactorsOfEachCar', (expect) => {
-    const expected = true;
-
-    const actual = true;
-
-    expect.equal(actual, expected);
+test("getAllCoolFactorsOfEachCar", (expect) => {
+  const expected = {
+    "Alfa Romeo": [
+      4, 9, 4, 4, 6, 2, 9, 8, 2, 2, 4, 5, 1, 4, 6, 10, 3, 7, 8, 9, 3, 9, 6, 2,
+      5, 3, 1, 10, 8, 1, 8, 10, 2, 8, 2, 10, 5, 1, 1, 8, 5, 3, 2, 7, 2,
+    ],
```

Comment:
This is essentially the same problem as the gender breakdown of each car. You are getting every customers cool factors and then setting them as the value for each car's key and instead we want to see the cool factors of the customers who have alfa romeo's, etc.. The array lengths of each car will be much shorter than this, some of them will have 1 and some will have 4, but they won't have 50ish